elements/bulbs-video/bulbs-video.test.js

```diff
@@ -6,15 +6,20 @@ describe('<bulbs-video>', () => {
   let subject;
   let props = {
     src,
+    disableLazyLoading: true,
   };
 
   beforeEach(() => {
     BulbsVideo.prototype.setState = sinon.spy();
     fetchMock.mock(src, {});
-    subject = new BulbsVideo(props);
```

Comment:
Moved this into each context that needs it since we want to delay this instantiation in lazy load tests.